src/cli/serve/serve.js

```diff
@@ -140,23 +140,12 @@ function applyConfigOverrides(rawConfig, opts, extraCliOptions) {
   }
 
   set('plugins.scanDirs', _.compact([].concat(get('plugins.scanDirs'), opts.pluginDir)));
-
   set(
     'plugins.paths',
     _.compact(
       [].concat(
         get('plugins.paths'),
         opts.pluginPath,
-        opts.runExamples
-          ? [
-              // Ideally this would automatically include all plugins in the examples dir
-              fromRoot('examples/demo_search'),
-              fromRoot('examples/search_explorer'),
-              fromRoot('examples/embeddable_examples'),
-              fromRoot('examples/embeddable_explorer'),
-            ]
-          : [],
```

Comment:
@stacey-gammon FYI you won't have to manually add these anymore

Reply:
awesome, thanks!!